tests/test_classifier/test_classification_report.py

```diff
@@ -203,11 +203,18 @@ def test_quick_method(self):
             random_state=27,
         )
 
+        # Create train/test splits
+        splits = tts(X, y, test_size=0.2, random_state=42)
+        X_train, X_test, y_train, y_test = splits
+        
         _, ax = plt.subplots()
         model = DecisionTreeClassifier(random_state=19)
-        classification_report(model, X, y, ax=ax, random_state=42)
+        visualizer = classification_report(
+            model, X_train, y_train, X_test, y_test, ax=ax, show=False
+        )
 
-        self.assert_images_similar(ax=ax, tol=25.0)
+        assert isinstance(visualizer, ClassificationReport)
+        self.assert_images_similar(visualizer)
```

Comment:
@Kautumn06 took a look at the image comparison failure on Travis and it looks like we need to add an image comparison tolerance to line 217. Since the Travis Miniconda tests are failing with `yellowbrick.exceptions.ImageComparisonFailure: images not close (RMS 11.995)`, I propose that we try updating this line to be: 

```python
self.assert_images_similar(visualizer, tol=12)
```

... and we'll see if that works! 🤞 🙏 😅

Reply:
No problem @rebeccabilbro! I pushed the changes so fingers crossed! However, if Travis is still unhappy, just let me know what else you'd like me to try!